src/sql/workbench/parts/jobManagement/browser/notebooksView.component.ts

```diff
@@ -981,4 +983,26 @@ export class NotebooksViewComponent extends JobManagementView implements OnInit,
 		let ownerUri: string = this._commonService.connectionManagementService.connectionInfo.ownerUri;
 		await this._commandService.executeCommand('agent.openNotebookDialog', ownerUri);
 	}
+
+	public async openLatesNotebookRun(notebook: azdata.AgentNotebookInfo) {
```

Comment:
should this be `openLatestNotebookRun` (missing a `t`)?

Reply:
Yeah, will make the changes